lib/facebook/messenger/server.rb

```diff
@@ -122,7 +144,7 @@ def trigger_events(events)
         events['entry'.freeze].each do |entry|
           # Facebook may batch several items in the 'messaging' array during
           # periods of high load.
-          entry['messaging'.freeze].each do |messaging|
+          entry['messaging'.freeze].try(:each) do |messaging|
```

Comment:
Do we need this `try`? We didn't before, and it's only available with ActiveSupport.